### PR TITLE
feat: add Cloud Run system packages example

### PR DESCRIPTION
**Pull request by ace-n**
```release-note:none
```

### CODE REVIEW

**Comment by ace-n** [comment]
Looks like the "graphviz" Docker image is not being provisioned, and that's causing the test failure. (See the `gcloud builds submit` command used [in the tutorial](https://cloud.google.com/run/docs/tutorials/system-packages#shipping_the_code)).

@pattishin @c2thorn how should we approach this? Should we:
1) provision that as part of the test?
2) use some sort of pre-built public image?

**Comment by c2thorn** [comment]
> provision that as part of the test?

@ace-n is that something that could be done in the same config? If yes, then that seems like a solid approach.

Otherwise, we could separate out the image into a doc-value and a test-value, using the test-value as a prebuilt.

**Comment by ace-n** [comment]
I _think_ it's possible using [this custom Terraform provider](https://github.com/kreuzwerker/terraform-provider-docker), but I have mixed feelings about that.

@pattishin WDYT?

**Comment by pattishin** [comment]
> I _think_ it's possible using [this custom Terraform provider](https://github.com/kreuzwerker/terraform-provider-docker), but I have mixed feelings about that. _(I'm wary of adding semi-random, non-official dependencies. But it may be better than having to set up a CD pipeline to continually rebuild this image.)_
> 
> @pattishin WDYT?

We shouldn't use custom Terraform provider. Is there a public graphiz image url we can leverage instead?

**Comment by ace-n** [comment]
@pattishin no - but (as we decided offline) we can:
1) use a standard `Hello World` image
2) tell users to replace that URL with one pointing to their `graphviz` container (via code comments and docs)

**Review by pattishin — CHANGES_REQUESTED** [review]

File: mmv1/products/cloudrun/terraform.yaml [hunk 1]
```diff
@@ -252,6 +252,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           cloud_run_service_name: "cloudrun-srv"
         min_version: beta
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_run_system_packages"
+        primary_resource_type: "google_cloud_run_service"
+        primary_resource_id: "default"
```
Comment (pattishin) [inline mmv1/products/cloudrun/terraform.yaml:258]: `skip_docs: true` So our quickstart snippets don't move up to the registry.hashicorp site.

**Review by pattishin — CHANGES_REQUESTED** [review]

File: mmv1/products/cloudrun/terraform.yaml (hunk 1 above)
Comment (ace-n) [reply mmv1/products/cloudrun/terraform.yaml:258]: Done.

(I think I missed this in previous PRs though - should I fix those?)

File: mmv1/templates/terraform/examples/cloud_run_system_packages.tf.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Example of how to deploy a Cloud Run application with system packages
```
[the remark this answers is not in the document]
Comment (ace-n) [reply mmv1/templates/terraform/examples/cloud_run_system_packages.tf.erb:1]: Whoops - done.

File: mmv1/products/cloudrun/terraform.yaml (hunk 1 above)
Comment (pattishin) [reply mmv1/products/cloudrun/terraform.yaml:258]: yes please 👍🏼

Comment (ace-n) [reply mmv1/products/cloudrun/terraform.yaml:258]: Done, see [this PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/6213/files).

**N.B:** I covered _all_ the Cloud Run samples there, as I didn't think you referring to the ones I wrote in isolation. 😄

**Review by pattishin — COMMENTED** [review]
lgtm

File: mmv1/products/cloudrun/terraform.yaml [hunk 3]
```diff
@@ -252,6 +252,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           cloud_run_service_name: "cloudrun-srv"
         min_version: beta
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_run_system_packages"
+        primary_resource_type: "google_cloud_run_service"
```
[the remark this answers is not in the document]
Comment (ace-n) [reply mmv1/products/cloudrun/terraform.yaml]: Done.

**Pull request merged**